app/controllers/api_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ApiController < ApplicationController
```

Comment:
Also this file should have been moved to the /api folder as a base class for all the api controllers